connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -3135,17 +3458,32 @@ ble_error_t Gap::createSync(
     sync_timeout_t timeout
 )
 {
+    tr_info("Create sync - "
+            "peerAddressType=%s, "
+            "peerAddress=%s, "
+            "sid=%u, "
+            "maxPacketSkip=%hu, "
+            "timeout=%lums",
+            to_string(peerAddressType),
+            to_string(peerAddress),
+            sid,
+            maxPacketSkip.value(),
+            timeout.valueInMs());
+
     if (is_extended_advertising_available() == false) {
+        tr_error("Failed: extended advertising not available");
         return BLE_ERROR_NOT_IMPLEMENTED;
     }
 
     if (peerAddressType != peer_address_type_t::PUBLIC &&
         peerAddressType != peer_address_type_t::RANDOM
         ) {
+        tr_error("Failed: peer address type must be public/random");
```

Comment:
There is a lot of variation on this message, probably 20 different strings, they could all be the same string:
```suggestion
        tr_error("can't use extended feature - controller doesn't support extended advertising");
```

Reply:
Is that not a little vague? `createSync` will fail unless the peer address is public/random. Therefore, to synchronize with periodic advertising, the user should ensure that this criteria is met.

Reply:
This comment is on the wrong line - sorry, no idea how this happened. This comment is meant for times when you check if extended advertising is present. This line is fine, yes.